game/Submods/NSFW Submod/nsfw_stories.rpy

```diff
@@ -272,63 +272,62 @@ label nsfw_erotic_story_thepen:
         call nsfw_story_begin
 
     if inspired_monika == True:
-        m 3eub "You remember when Yuri took the main character's pen and did some..."
-        m 1dublsdlu "Ahem..."
-        m 1rkblsdlb "Questionable things with it?"
+        m 3eub "Do you remember when Yuri mentioned how she took the main character's pen and did some...ahem..."
+        extend 1rkblsdlb "questionable things with it?"
         m 3eubla "Well, that's the inspiration for this story."
-    m 3eub "The other day while you were gone [player], I wanted to try something."
+    m 3eub "The other day while you were gone, I wanted to try something."
     m 3hksdlb "It might sound strange, but hear me out!"
-    m 3rubla "...I was in a horny mood..."
-    m 3rkblb "...And I felt the urge to masturbate."
+    m 3rubla "I was in a horny mood..."
+    m 3rkblb "And I felt the urge to masturbate."
     if renpy.seen_label("nsfw_sexting_finale"):
-        m 3ekblb "But I didn't want to just play with myself like we did when we sexted."
+        m 3ekblb "But, I didn't want to just play with myself with my fingers like we did when we sexted."
     else:
-        m 3ekblb "But I didn't want to just play with myself using my fingers."
+        m 3ekblb "But, I didn't want to just play with myself using my fingers."
     m 2esbla "I wanted to try something {i}different{/i}, you know?"
-    m 2eub "So...I was in my room, searching for something that will make masturbating more exciting."
-    m 3eua "When I found a pen sitting on the desk."
+    m 2eub "So...I was in my room, searching for something that would make it more exciting..."
+    extend 3eua "when I found my pen sitting on the desk."
     if inspired_monika == True:
-        m 3rkblb "My mind immediately went back to when I overheard that Yuri had used the main character's pen when she masturbated."
-        m 3rtblu "I wondered if it was that good of a feeling, or if she was just so obsesseed with the main character that anything would have done it."
+        m 3rkblb "My mind immediately went back to when I overheard that Yuri had used the main character's pen to stimulate herself."
+        m 3rtblu "I wondered if it was that good of a feeling, or if she was just so obsessed with him that anything would have done it."
     else:
         m 3rkblb "My mind immediately filled with thoughts of using it."
     m 1hublb "Ahaha! I can't see your face, but I can already tell you're asking why, right?"
     m 1eubla "Well, I understand you might think it's strange."
     m 3tubla "But when you don't have anything but your fingers to masturbate with..."
-    m 3hublb "You get kind of desperate. Ahaha~"
+    extend 3hublb "you get kind of desperate. Ahaha~"
     m 4eublb "Anyway, I took the pen and walked back to the bed."
-    m 4tublb "I first stripped down to my underwear, and then laid down on the bed."
+    m 4tublb "I first stripped down to my underwear, and then lay down."
```

Comment:
Is this grammar correct? Saying 'lay down' sounds more like a command than something she did herself.

Reply:
It sounds a bit weird but[ _lay_ is the past tense of _lie_](https://www.britannica.com/story/lay-lie-lied-lain-when-do-we-use-which), a word describing an intransitive action that you perform yourself. _Laid_ is the past tense of _lay_ (a different "lay"!), a transitive word describing setting something else down. 

It is technically grammatically correct but I agree it sounds odd. Maybe we should change it outright to 
`m 4tublb "First I stripped down to my underwear, and then I reclined back on my bed and made myself comfortable."`?

Reply:
Yeah I reckon that will work best. Sounds like a huge headache otherwise lol